rpcs3/util/sysinfo_darwin.mm

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
```

Comment:
This belongs in a different directory. iirc xcode stuff is objective-c/c++ right?

Reply:
Yes. This file is Objective-C++ (mixed Objective-C + C++).